### PR TITLE
M13467 documentation 1

### PR DESCRIPTION
**Pull request by m13467**
This has the updated README with the gifs, byoLAD logo, and contribute.md.

### CODE REVIEW

**Review by EianWright — COMMENTED** [review]
@byubean, is this one that you want to review?

Also, when this is merged, can we do "squash and merge" instead of the standard merge?

File: README.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
-# byoLAD
+<img src="byo_LAD.png"  width="200" height="200">
+
+# byoLAD: Your code improvement buddy
 
 byoLAD augments your development experience by reviewing your code, identifying errors, adding documentation, and suggesting general improvements.  Since byoLAD is open-source, you can modify prompts and other configurations to your liking. 
 
-In addition, byoLAD enables you to use your own LLM API, so you can use your favorite AI and control how much you spend. We don't add any extra costs, and there are no extra parties looking at your data. You only have to manage your account with your LLM provider. 
+In addition, byoLAD enables you to use your own Large Language Model (LLM) API, so you can use your favorite AI and control how much you spend. We don't add any extra costs, and there are no extra parties looking at your data. You only have to manage your account with your LLM provider.
```
Comment (byubean) [inline README.md:7]: "You ~only have to~ manage your account directly with your LLM API provider."

File: README.md [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
 
-In addition, byoLAD enables you to use your own LLM API, so you can use your favorite AI and control how much you spend. We don't add any extra costs, and there are no extra parties looking at your data. You only have to manage your account with your LLM provider. 
+In addition, byoLAD enables you to use your own Large Language Model (LLM) API, so you can use your favorite AI and control how much you spend. We don't add any extra costs, and there are no extra parties looking at your data. You only have to manage your account with your LLM provider. 
 #### LLM Providers Currently Supported: 
-- OpenAI
-- Google
-
+- [OpenAI](https://openai.com/blog/openai-api)
+- [Google](https://developers.generativeai.google/)
+
```
Comment (byubean) [inline README.md:11]: Let's add sub-bullets for the models and API token instruction pages for each provider.

File: bean.jpg
Comment (byubean) [inline bean.jpg:1]: Let's remove this from the repo.

**Review by byubean — APPROVED** [review]
Address comments in another PR.

**Pull request merged**